### ISSUE TITLE
mObywatel crashes

### ISSUE THREAD

**Issue by mateusz-holysz** [GrapheneOS/os-issue-tracker#3773]
mObywatel is polish official app released by government for digital documents such as e-id, driving licence, right to practise etc. 

https://play.google.com/store/apps/details?id=pl.nask.mobywatel

mObywatel 4.37.0 (2508)  is constantly crashing on start. Previous versions worked fine, without issues.

My compilation version is 2024071699
My device is Pixel 6a

mObywatel logs attached
[Error in mObywatel 882d1956d241.txt](https://github.com/user-attachments/files/16316914/Error.in.mObywatel.882d1956d241.txt)
[mObywatel log 662c3fb33326.txt](https://github.com/user-attachments/files/16316915/mObywatel.log.662c3fb33326.txt)

**Comment by mateusz-holysz** [comment]
Yes. I have tried to enable exploit protection compatibility mode. Fully and in a variety of combinations. Do you need logs with this option enabled?

Kind regards

Mateusz Hołysz


19 lip 2024, 22:51 od ***@***.***:

**Comment by FID02** [comment]
I can't reproduce on a Pixel 8 nor on a Pixel 6a running the latest OS version and version of the app. Not even when launched without Play Services installed. But I don't have an account so I was only able to launch it and reach the registration wizard.

Is the app crashing upon each launch of the app? 

Could you try the following steps to see if it resolves the issue?
1. Go to Settings > Security > Secure app spawning: disable it
2. Disable exploit protection compatibility mode for mobywatel
3. Reboot the phone
4. Open mobywatel

**Comment by mateusz-holysz** [comment]
Yes, app crashing every time I try to launch it.
Disabling secure app spawning, disabling (enabling) exploit protection compatibility mode for app and proceeding reboots results nothing. Still not available to launch app.

Thank You

21 lip 2024, 12:48 od ***@***.***:

**Comment by mateusz-holysz** [comment]
I have tried to reinstall. Now it works fine. Even is quite hard to restore all needed digital documents, but it certainly works. Just courius, what prevented this app to launch properly? Nothing changed instead reinstall.

Thank You 

21 lip 2024, 19:06 od ***@***.***:
